src/hooks/useFetch.js

```diff
@@ -30,7 +30,7 @@ export default function useFetch({
 }) {
   const [displayedError, setDisplayedError] = useState(null);
   const [displayedLoading, setDisplayedLoading] = useState(
-    !queryOptions.disabled,
+    !queryOptions.disabled, // should this use enabled instead of disabled? I couldn't find anything in the react-query documentation about disabled.
```

Comment:
It looks like it was added in #330 when `config.disabled` was added to some of the useFetch derived hooks. Even those hooks set `queryOptions.enabled` based on that config, so it doesn't look like `queryOptions.disabled` is ever set to true. Please correct me on this if you find a case where it is.

That means that this line doesn't need to be `queryOptions.enabled`, but can just default to `true` since that is how it always currently evaluates anyway.

Reply:
I also didn't find anything anywhere else.
I'm still a bit confused about what this does in practical terms, but I'll pull my comment out in the meantime.